Kickstarter-iOS/DataSources/PledgeDataSource.swift

```diff
@@ -34,8 +34,8 @@ final class PledgeDataSource: ValueCellDataSource {
     }
 
     self.appendRow(
-      value: Strings.Total(),
-      cellClass: PledgeRowCell.self,
+      value: (project, 240, 25),
```

Comment:
Hard-coded for now, will handle this as part of handling the state changing between cells.